lib/cli.rb

```diff
@@ -150,7 +150,8 @@ def self.check_container_name!(image, name)
         "Error: System description name '#{name}' is invalid. By default Machinery" \
           " uses the image name as description name if the parameter `--name` is not" \
           " provided.\nIf the image name contains a slash the `--name=NAME` parameter" \
-          " is mandatory. Valid characters are 'a-zA-Z0-9_:.-'."
+          " is mandatory. Valid characters are 'a-zA-Z0-9_:.-'.\n\nFor example run:" \
```

Comment:
A linebreak after `run:` would be helful so that you can directly copy the whole line.